### ISSUE TITLE
Focus on element using keyboard with tooltip is not shown

### ISSUE THREAD

**Issue by matuzalemsteles** [liferay/clay#5153]
When the user is moving through elements on a page using the keyboard and some elements have the tooltip defined, when focusing on these elements the tooltip is not shown.

The important part here is that, there is a difference between the focus with the keyboard and the focus when clicking on the element, clicking on the element should not show the tooltip, it should follow the rule of showing only after 500ms on top of the element, when using the keyboard the tooltip should be shown.

**Comment by matuzalemsteles** [comment]
cc @drakonux @emiliano-cicero maybe we can add this behavior in the document or just link to https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/.

**Comment by edalgrin** [comment]
> cc @drakonux @emiliano-cicero maybe we can add this behavior in the document or just link to https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/.

Also, consider these 2 points from this other documentation https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role
<img width="795" alt="image" src="https://user-images.githubusercontent.com/46778114/197781158-28aaa7e5-60e3-4363-9469-bc11792eed71.png">

**Comment by emiliano-cicero** [comment]
hey @matuzalemsteles, my knowledge about this is that if there's a tooltip, the screen reader should read it when there's a focus on the elements using the keyboard.
I also agree with @edalgrin that the tooltips should be used only if the information is not essential.

And I agree that this should be in the documentation: could you open a Lexi request in Jira so that we keep an eye on it? Thanks!
